pkg/apis/awsprovider/v1alpha1/awsmachineproviderconfig_types.go

```diff
@@ -37,6 +37,9 @@ type AWSMachineProviderSpec struct {
 	// AMI is the reference to the AMI from which to create the machine instance.
 	AMI AWSResourceReference `json:"ami,omitempty"`
 
+	// ImageLookupOrg is the AWS Organization ID to use for image lookup if AMI is not set.
+	ImageLookupOrg string `json:"imageLookupOrg,omitempty"`
```

Comment:
Nit, ImageLookupOrg could be `ImageLookupOrgID`

Reply:
I had thought about that, but I also felt it was already a bit long.